src/main/java/org/springframework/social/bitbucket/api/impl/EmailsTemplate.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (C) 2012 Eric Bottard / Guillaume Lederrey (eric.bottard+ghpublic@gmail.com / guillaume.lederrey@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.social.bitbucket.api.impl;
+
+import org.springframework.social.bitbucket.api.BitBucketEmailAddress;
+import org.springframework.social.bitbucket.api.EmailsOperations;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author Cyprian Śniegota
+ * @since 2.0.0
+ */
+public class EmailsTemplate extends AbstractBitBucketOperations implements EmailsOperations {
+
+    public EmailsTemplate(RestTemplate restTemplate, boolean authorized) {
+        super(restTemplate, authorized, V1);
+    }
+
+    @Override
+    public final List<BitBucketEmailAddress> getListOfUserEmailAddresses(String accountName) {
+        return Arrays.asList(getRestTemplate().getForObject(buildUrl("/users/{accountname}/emails"),
```

Comment:
I would add a static import for `Arrays.asList()`. (minor comment, feel free to ignore).